### PR TITLE
Chat Console

### PR DESCRIPTION
**Pull request by Deimosu**
Fix tables etc. on DE:Chat Console
Uploaded images, also uploaded supporter_tag.png to shared folder

### CODE REVIEW

**Comment by Deimosu** [comment]
2 new commits:
deleting supporter_tag.png in shared (osu!supporter.png exists)
updated the de.md file to use the correct image

**Comment by Deimosu** [comment]
added image (user panel options in english) and fixing tables in Chat Console/en.md, missing images etc.

**Comment by Deimosu** [comment]
stupid <> chars in tables, replaced them with [] in Chat Console/de.md

**Comment by Deimosu** [comment]
In short: 3 new commits (rest is redundant)

- updated fr.md
- uploaded image for userpanel options
- fixed bancho repsonse for '/delfriend [user]'

File: wiki/Chat_Console/en.md [hunk 1]
```diff
@@ -3,21 +3,21 @@ Basic Chat Console
...
+|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| White  | You                                                                                                                                                                                                      |
+| Pale   | Non-supporters                                                                                                                                                                                           |
+| Yellow | ![Supporter Tag](/wiki/shared/osu!supporter.png "Supporter Tag") Supporters                                                                                                                                            |
+| Red    | [Global Moderation Team](/wiki/People/Global_Moderation_Team/en.md "Global Moderation Team") or [Quality Assurance Team](/wiki/People/Quality_Assurance_Team/en.md "Quality Assurance Team") member      |
```
Comment (RaikireHiuduo) [inline wiki/Chat_Console/en.md]: Replace all instance of `/*.md` (en.md, de.md, any with .md extension).

The osu-web will do the proper direction by itself. 

So, for this line:-
```Markdown
[Global Moderation Team](/wiki/People/Global_Moderation_Team/ "Global Moderation Team") or [Quality Assurance Team](/wiki/People/Quality_Assurance_Team/ "Quality Assurance Team") member
```
(see the missing `en.md`?)

File: wiki/Chat_Console/en.md [hunk 2]
```diff
@@ -3,21 +3,21 @@ Basic Chat Console
...
+| Green  | Line contains your name or certain keywords that you have added to trigger a "highlight". A copy of the message will also appear in a dedicated channel, \#highlight, which contains all of these lines. |
+| Blue   | Private Message                                                                                                                                                                                          |
+| Cyan   | [peppy](https://osu.ppy.sh/u/2 "peppy"), the creator of osu!.                                                                                                                                            |
+| Pink   | [BanchoBot](/wiki/BanchoBot/de.md "BanchoBot")                                                                                                                                                           |
```
Comment (RaikireHiuduo) [inline wiki/Chat_Console/en.md]: remove `de.md`.

File: wiki/Chat_Console/en.md [hunk 3]
```diff
@@ -26,12 +26,12 @@ From most screens in osu! you can press F8 or click the "Show Chat" button on th
 Extended Chat Console
 =====================
 
-*[osu!academy](osu!academy "wikilink") covered this page in [Episode 6 (6:52)](http://www.youtube.com/watch?v=cyYRl-a5xII) along with [Multiplayer](Multiplayer "wikilink").*
+*[osu!academy](/wiki/Announcements/osu!academy/ "osu!academy") covered this page in [Episode 6 (6:52)](http://www.youtube.com/watch?v=cyYRl-a5xII) along with [Multiplayer](/wiki/Multi/en.md "Multiplayer").*
```
Comment (RaikireHiuduo) [inline wiki/Chat_Console/en.md]: ditto

**Pull request closed without merging**